rollup.config.js

```diff
@@ -2,15 +2,26 @@ import commonjs from "@rollup/plugin-commonjs";
 import nodeResolve from "@rollup/plugin-node-resolve";
 const dependencies = require("./package.json").dependencies;
 
-export default Object.keys(dependencies).map((depName) => {
-  return {
-    input: `node_modules/${depName}`,
+export default Object.keys(dependencies)
+  .map((depName) => {
+    return {
+      input: `node_modules/${depName}`,
+      output: {
+        file: `src/fontra/client/third-party/${depName}.js`,
+        inlineDynamicImports: true,
+      },
+      plugins: [
+        commonjs(), // <-- this handles some parsing of js syntax or something (necessary for `export { init } from "mathjax";`)
+        nodeResolve({ browser: true }), // <-- this allows npm modules to be added to bundle
+      ],
+    };
+  })
+  .concat({
+    input: [
+      "node_modules/lib-font/lib/inflate.js",
```

Comment:
I'm not really happy with this special case for lib-font here, but I do not yet have a concrete proposal how to do it differently. Once I do, I will suggest a followup task.

Reply:
This is a special case about how the lib-font is bundled. I am not happy with that, at all.